core/tests/test_utils.py

```diff
@@ -558,6 +558,26 @@ def delete_json(self, url, params='', expect_errors=False,
     def _send_post_request(
             self, app, url, data, expect_errors=False, expected_status_int=200,
             upload_files=None, headers=None):
+        """Sends a post request with the data provided to the url specified.
+        Args:
+            app: WSGI application. The WSGI application which receives the request
+                and produces response.
+            url: str. The URL to send the POST request to.
+            data: To be put in the body of the request. If params is an iterator,
```

Comment:
In type info, you can add `*` which indicates that this arguments has varying types.